data/changelog.txt

```diff
@@ -300,6 +311,7 @@ Improves performance of initializing the achievements tracker.
 When NPCs are fleeing from explosions, skips considering routing through tiles without a floor.
 Speeds up fungus-related code processing by 30%+.
 Optimizes item migrations to decrease load times.
+Fixes bad performance on large screens with lots of text (especially overmap).
```

Comment:
```suggestion
Restores graphics speed for large screens with lots of text.
```
Since this PR didn't actually improve FPS over 0.E-3, as far as I can tell, but instead restores it to that level - should it go into Bugfixes, not Performance?